js/api-augment/substrate_v1_rpcs.ts

```diff
@@ -0,0 +1,75 @@
+import { DefinitionRpc } from '@polkadot/types/types';
+
+export const v1SubstrateRpcs: Record<string, Record<string, DefinitionRpc>> = {
```

Comment:
Is this generated? If not what is the source and how it might evolve in future?

Reply:
It is not. I generated it via the list of "unknown" rpcs. I went looking for being able to pull it from somewhere, but nothing had it from what I could find on GitHub. :(